### ISSUE TITLE
Dwarf witch tile

### ISSUE THREAD

**Issue by davidk64fnq** [NickMcConnell/FAangband#320]
I came across one and it displayed as a purple "p". In the monster list there is an entry for witch but not dwarf witch which makes me think the code assigns a random race to the witch monster for each instance?

If the code uses a single tile for all instances of witch that is problematic? 

Perhaps something along the lines if monster doesn't have a tile assigned use a generic race tile? 

Were it other than a dwarf you might even have two generic tiles for each race distinguishing sex?

**Comment by NickMcConnell** [comment]
Yes, "player race monsters" (they get the PLAYER flag in monster.txt) have a player race attached.  I think on the whole it's best to have a tile for witch rather than for the player race; all witches have the same spells, HP, etc, whereas the only real effect of the player races is how likely they are to be hostile to the player.
